src/proto/feature_flag.proto

```diff
@@ -41,4 +41,8 @@ package otbr;
 message FeatureFlagList {
   // Whether to enable the NAT64 feature.
   optional bool enable_nat64 = 1 [default = false];
+  // Whether to enable detailed logging.
+  optional bool enable_detailed_logging = 2 [default = false];
+  // Set specific detailed logging level, default = OTBR_LOG_INFO
```

Comment:
Can you add detail comment about which number corresponds to which LOG level?

Reply:
Done

Reply:
If the OTBR's initial log level is INFO, when you enable detailed logging, do you want to use same INFO level? Or you want to use detailed debug level?

Reply:
Normally the user build images have log level NOTE, setting log level to INFO on these devices would help with debugging. INFO level might already be sufficient, if we figured DEBG is more desirable we can use the 'detailed_logging_level' field to change it to DEBG.